lib/tasks/configure_solr.rake

```diff
@@ -1,36 +1,27 @@
 require 'rails/generators'
 require 'generators/geoblacklight/install_generator'
 require 'geoblacklight'
-require 'open-uri'
+require 'fileutils'
 
 namespace :geoblacklight do
-  desc 'Copies the default SOLR config for the included Solr'
-  task :configure_solr do
-    files_urls = [
+  desc 'Configures Solr for local jetty instance'
+  task :configure_solr, [:version] do |_t, args|
+    args.with_defaults(version: '4.7')
+    [
       {
-        url: 'https://raw.githubusercontent.com/geoblacklight/geoblacklight-schema/v0.3.0/conf/schema.xml',
+        src: "schema/solr/#{args[:version]}/conf/schema.xml",
```

Comment:
Looks like it does break. Maybe something like:

`Gem::Specification.find_by_name("your_gem_name").gem_dir`

or 

http://stackoverflow.com/a/10133837

I think you may have `Blacklight.root` available to you?